src/sandbox/enviroments/base/env.py

```diff
@@ -0,0 +1,22 @@
+from typing import Iterable, Optional
+import gym
+from gym.spaces.space import Space
+from sandbox.enviroments.base.action import Action
+
+from sandbox.enviroments.base.observation import Observation
+
+class Environment(gym.Env):
```

Comment:
I think it would be better to create our wrapper for `gym,Env`, so we will have more control over it.

Reply:
Pro: it may be easier for students.
Contra: students won't learn gym.Env

@czyjtu what do you think?

Reply:
I'd rather use `gym.Wrapper` for that purpose. 
Check out exemplary wrappers from https://github.com/openai/gym/tree/master/gym/wrappers

Reply:
IMHO, wrappers look nice